lib/gtfs/components/GtfsFilter.js

```diff
@@ -64,50 +63,48 @@ export default class GtfsFilter extends Component {
           </Dropdown.Toggle>
           <Dropdown.Menu className='scrollable-dropdown'>
             {allFeeds.length > 0
-             ? allFeeds.map((feed) => {
-               const isPublished = feed.publishedVersionId !== null
-               const disabled = loadedFeeds.findIndex(f => f.id === feed.id) === -1
-               return (
-                 <MenuItem
-                   key={feed.id}
-                   style={{width: '200px'}}
-                   eventKey={feed.id}
-                   alt={feed.shortName || feed.name}
-                   disabled={disabled}
-                   title={feed.shortName || feed.name}>
-                   <span>
-                     <Label
-                       bsStyle={disabled && isPublished
-                         ? 'warning'
-                         : disabled
-                         ? 'danger'
-                         : activeFeeds.indexOf(feed) === -1
-                         ? 'danger'
-                         : 'success'
-                       }
-                       className='pull-right'>
-                       {disabled && isPublished
-                         ? 'pending'
-                         : disabled
-                         ? 'unpublished'
-                         : activeFeeds.indexOf(feed) === -1
-                         ? 'off'
-                         : 'on'
-                       }
-                     </Label>
-                     <span>
-                       {feed.shortName || feed.name.length > 11
-                         ? feed.name.substr(0, 11) + '...'
-                         : feed.name
-                       }
-                     </span>
-                   </span>
-                 </MenuItem>
-               )
-             })
-            : <MenuItem disabled>
-             No feeds available
-            </MenuItem>
+              ? allFeeds.map((feed) => {
+                const isPublished = feed.publishedVersionId !== null
+                const disabled = loadedFeeds.findIndex(f => f.id === feed.id) === -1
```

Comment:
Rewrite this using some() so that the search doesn't have to go through all feeds.

Ex:
```javascript
!loadedFeeds.some(f => f.id === feed.id)
```

Reply:
Can you explain why this is more efficient than [`Array#findIndex`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/findIndex#Description)?  Don't they both immediately return once they find a truthy value?